Mongo DB/mongodb.md

```diff
@@ -0,0 +1,17 @@
+##OVERVEIW:<br><br>
+
+MongoDB is a free and open-source cross-platform document-oriented database program (stores data as a "document" inside a collection) written in C++.MongoDB is a leading NoSQL database and an open source document database.MongoDB works on terminologies which includes database,collection and document. It can be used in Bigdata,Datahub,Mobile Infrastructure Development,User Data Management. It works on the concept of document and collection. It is currently used by thousands of top organizations like Source Forge, Craigslist,eBay, Viacom, Foursquare and The New York Times.<br><br>
+One of the most extinguishing features of MongoDB is the process of sharding. It is the process of storing data records across multiple machines.Upon increasing the data size a single machine may not be able to store data,here comes the concept of sharding. It's an attempt of MongoDB to meet the demands of data growth.<br><br>
+###OTHER FEATURES OF MongoDB:<br>
+  Document-oriented<br>
+  Indexing<br>
+  Replication<br>
+  File storage<br>
+  Server-side JavaScript execution<br>
+  Capped collections<br><br>
+###ADVANTAGES OF USING MongoDB:<br>
+   One does'nt need much time to design database.<br>
+   It is scalable and consistent in nature.<br>
+   It offers us freedom to run anywhere.<br>
+
+[MongoDB INSTALLATION GUIDE.](https://docs.mongodb.com/manual/installation/)

```

Comment:
Take reference from the guide over how to install and use it. Little bit of more usage and less  theory will be preferred